roles/lib_utils/filter_plugins/oo_filters.py

```diff
@@ -660,6 +660,42 @@ def map_from_pairs(source, delim="="):
     return dict(item.split(delim) for item in source.split(","))
 
 
+def lib_utils_oo_get_node_labels(source, hostvars=None):
+    ''' Return a list of labels assigned to schedulable nodes '''
+    labels = list()
+
+    # Filter out the unschedulable nodes
+    for host in source:
+        if host not in hostvars:
+            return
+        node_vars = hostvars[host]
+
+        # All nodes are considered schedulable,
+        # unless explicitly marked so
+        if node_vars.get('openshift_schedulable') is False:
```

Comment:
Need to implement something similar to the to_bool function here: roles/lib_utils/action_plugins/sanity_checks.py

User could provide 'false' as value in ini inventory variable, that would resolve to a string type, and this error would be hard to track down later.

Reply:
Good idea, there is already `strtobool` from distutils imported there, it should do the trick